runtime/vm/FFITypeHelpers.cpp

```diff
@@ -23,6 +23,55 @@
 #include "FFITypeHelpers.hpp"
 
 #ifdef J9VM_OPT_PANAMA
+ffi_type*
+FFITypeHelpers::getFFITypeVararg(j9object_t vararg, void **varargValue)
+{
+	PORT_ACCESS_FROM_JAVAVM(_vm);
+
+	ffi_type* typeFFI = NULL;
+	J9Class *varargClass = J9OBJECT_CLAZZ(_currentThread, vararg);
+
+	if (varargClass == J9VMJAVALANGINTEGER_OR_NULL(_vm)) {
+
+		U_32 *value = (U_32 *)j9mem_allocate_memory(sizeof(U_32), OMRMEM_CATEGORY_VM);
```

Comment:
Instead of mallocing, create an array (stack allocate and malloc if not big enough) for all the varags values and assign the `varargValue` pointers to that